### DISCUSSION TITLE
Javascript version for cinnamon-spices

### DISCUSSION THREAD

**Discussion by mortim** [linuxmint/discussions#464] [Questions] [answered]
Hi,
I was wondering why cinnamon spices (like applets, extensions, desklets, etc...) are written in Javascript ES5 and not ES6 (the syntax is clearer for me).

**Comment by rcalixte** [comment, marked as answer]
Both ES5 and ES6 are supported in Cinnamon Spices. Each Spice is maintained and therefore directed by the individual maintainer for each. If you take a look, some have been updated to ES6 while others, as you pointed out, are ES5. If you prefer to write or maintain something in ES6, you should feel encouraged to do so.
